gatsby/plugins/gatsby-source-cks/src/api/api.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Low-level methods to access each of the CKS API endpoints
+ *
+ * @module
+ */
+
+import fetch from "node-fetch";
+import moment from "moment";
+import { Cache } from "gatsby";
+
+import {
+	ApiConfig,
+	ApiTopicsResponse,
+	ApiTopicResponse,
+	ApiTopicChangeResponse,
+	ApiPartialTopic,
+	ApiFullTopic,
+} from "./types";
+
+export const apiConfig: ApiConfig = {
+	apiBaseUrl: "",
+	apiKey: "",
+};
+
+export const configure = ({ apiBaseUrl, apiKey }: ApiConfig): void => {
+	apiConfig.apiBaseUrl = apiBaseUrl;
+	apiConfig.apiKey = apiKey;
+};
+
+const verifyConfig = (): true => {
+	if (
+		!apiConfig.apiBaseUrl ||
+		apiConfig.apiBaseUrl === "" ||
+		!apiConfig.apiKey ||
+		apiConfig.apiKey === ""
+	)
+		throw new Error(
+			"Please call configure with a base url and key before using the API."
+		);
+
+	return true;
+};
+
+/**
+ * Wrapper function around fetch that addds headers for
+ * authentication and JSON accept content type
+ */
```

Comment:
you are using && here instead of ; to enable usage of shorthand syntax, neat!

Reply:
Thanks! Shorthand but still readable/understandable I hope